contrib/devtools/update-translations.py

```diff
@@ -52,6 +52,10 @@ def find_format_specifiers(s):
         percent = s.find('%', pos)
         if percent < 0:
             break
+        try:
+            specifiers.append(s[percent+1])
+        except:
```

Comment:
Please no wildcard catches

Reply:
While sharing the vision of aiming for perfectionism, a proper "validation" of specifiers might be difficult, if going for more than a simple notification then I would also try to cover possible cases where the input data can be misinterpreted by missing a trailing space. I will think about a better while still easy solution and update the PR.

Reply:
IMO we should check those cases instead of catching the exception.

Reply:
translators accidentally like to sneak in a space after % as well, probably there's a solution to identify all of those errors. tbh I didn't think much about it recently

Reply:
I don't see what other error could be thrown here. Just catch the one you know the code should produce.

Reply:
you guys are absolutely right, I was thinking about doing some kind of auto-correct but better keep it simple but clean. sorry for not getting to it for a while